### PR TITLE
Add release create

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: cmd/releases.go [hunk 1]
```diff
@@ -51,3 +56,55 @@ func runReleases(ctx *cli.Context) error {
...
+		TagName:      ctx.String("tag"),
+		Target:       ctx.String("target"),
+		Title:        ctx.String("title"),
+		Note:         ctx.String("note"),
+		IsDraft:      ctx.Bool("draft"),
+		IsPrerelease: ctx.Bool("prerelease"),
+	})
+
+	if err != nil {
+		log.Fatal(err)
```
Comment (daviian) [inline cmd/releases.go:106]: Shouldn't the user be able to see the error message in cli?

File: cmd/releases.go [hunk 2]
```diff
@@ -51,3 +56,55 @@ func runReleases(ctx *cli.Context) error {
 
 	return nil
 }
+
+var CmdReleaseCreate = cli.Command{
+	Name:        "create",
+	Usage:       "Create a release on repository",
```
Comment (daviian) [inline cmd/releases.go]: I'd say `in repository`

File: cmd/releases.go (hunk 1 above)
Comment (lunny) [reply cmd/releases.go:106]: Currently log will output to console.

File: cmd/releases.go (hunk 2 above)
Comment (lunny) [reply cmd/releases.go]: done.

**Review by daviian — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**